src/test/java/tutorease/address/logic/commands/AddContactCommandTest.java

```diff
@@ -142,6 +142,16 @@ public boolean hasPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public boolean hasSamePhone(Person person) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public boolean hasSameEmail(Person person) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
I like the catch you did on how these methods should not be called within the other commands